packages/content/test/attributes/Website.test.ts

```diff
@@ -0,0 +1,73 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { Website } from "../../src";
+
+describe("creation of RepositoryAttributes of Attribute Value Type Website", function () {
+    test("can create a RepositoryAttribute of Attribute Value Type Website", function () {
+        const validWebsite = Website.from({
+            value: "https://inwind.it"
+        });
+        expect(validWebsite.value.toString()).toBe("https://inwind.it");
+    });
+    test("can create a RepositoryAttribute of Attribute Value Type Website with the German 'ä' in its domain", function () {
+        const validWebsite = Website.from({
+            value: "https://inwänd.it"
+        });
+        expect(validWebsite.value.toString()).toBe("https://inwänd.it");
+    });
+    test("can create a RepositoryAttribute of Attribute Value Type Website without www.", function () {
+        const validWebsite = Website.from({
+            value: "//google.it"
+        });
+        expect(validWebsite.value.toString()).toBe("//google.it");
+    });
+    test("can create a RepositoryAttribute of Attribute Value Type Website with a enhanced path", function () {
+        const validWebsite = Website.from({
+            value: "https://enmeshed.de/blog/meilenstein-enmeshed-als-komponente-ablage-in-mein-bildungsraum-geht-in-die-testphase-der-beta-version/"
+        });
+        expect(validWebsite.value.toString()).toBe(
+            "https://enmeshed.de/blog/meilenstein-enmeshed-als-komponente-ablage-in-mein-bildungsraum-geht-in-die-testphase-der-beta-version/"
+        );
+    });
+    test("returns an error when trying to create an Attribute Value Type Website with a blank in the value for Website", function () {
+        const invalidWebsiteCall = () => {
+            Website.from({
+                value: "Hugo https://inwind.it"
```

Comment:
Would and should this work if the blank wasn't in that place? `Hugohttps://inwind.it` I would have expected the blank somewhere within in the actual domain.

Reply:
It would never work if there would be a blank in any place. I was focused on the thing that would have been worked before and that was one of these things. But I can change the title of the test like: 
"returns an error when trying to create an Attribute Value Type Website which contains more than only the website"

Reply:
I actually preferred the name you had before, since it was more concrete.

Reply:
I rechanged it now.